Demo/Demo/AppDelegate.swift

```diff
@@ -5,7 +5,9 @@ import UIKit
 import PlayerControls
 
 typealias Props = ContentControlsViewController.Props
-func props() -> Props {
+
+func props(progress: ContentControlsViewController.Props.Progress? = nil) -> Props {
+    let progress = progress ?? 1
```

Comment:
Why not defined in func param?

Reply:
Why it is optional?

Reply:
@BogdanBilonog @AndriiMoskvin I've removed optional. I'm agreed, that it is not really necessary.